### PR TITLE
Review and cs for the other files in the install folder

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes

Code Review and Cs for the models folder of the installation
#### Testing Instructions

Confirm that the installation still works with this branch: https://github.com/zero-24/joomla-cms/archive/installcsother.zip

### CODE REVIEW

**Comment by zero-24** [comment]
@mbabker can you help me? Travis told me something i don't understand: https://travis-ci.org/joomla/joomla-cms/jobs/128932764

```
FILE: /home/travis/build/joomla/joomla-cms/installation/template/index.php
--------------------------------------------------------------------------------
FOUND 2 ERROR(S) AFFECTING 2 LINE(S)
--------------------------------------------------------------------------------
 52 | ERROR | Closing brace must be on a line by itself
 54 | ERROR | Closing brace must be on a line by itself
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------
```

I can't see any wrong brace there?!

**Comment by mbabker** [comment]
The PHPCS ruleset needs edited, the template's index.php file should be on the ignore list for whatever rule the other layouts have that allow that to work.  Or just revert it and leave it alone.

**Comment by zero-24** [comment]
Thanks i have just add it to the general roule to ignore that file @mbabker

File: build/phpcs/Joomla/ruleset.xml [hunk 1]
```diff
@@ -23,6 +23,8 @@
  <exclude-pattern type="relative">administrator/components/com_joomlaupdate/restore_finalisation.php</exclude-pattern>
  <exclude-pattern type="relative">administrator/components/com_joomlaupdate/restore.php</exclude-pattern>
  <exclude-pattern type="relative">configuration.php</exclude-pattern>
+ <exclude-pattern type="relative">configuration.php</exclude-pattern>
```
Comment (brianteeman) [inline build/phpcs/Joomla/ruleset.xml]: Why is this line twice

Comment (zero-24) [inline build/phpcs/Joomla/ruleset.xml]: Oops. Thats wrong... I can fix it tomorrow morning. Thanks

**Comment by zero-24** [comment]
Thanks @brianteeman i have just fixed / removed the extra line ;)

**Comment by pritalpatel** [comment]
I have tested this item :white_check_mark: successfully on a5ddef183520ec95cf00c799f97b75ae86c68b6a<br /><br />Installed using provided link. It works normally.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10363">issues.joomla.org/joomla-cms/10363</a>.</sub>

**Comment by zero-24** [comment]
Is this so hard to test? Please let me know what is unclear than I can add a better desciption ;)

**Comment by bertmert** [comment]
I have tested this item :white_check_mark: successfully on d5975afdc7e80b10e668058dd286e3b9d4aabcfd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10363">issues.joomla.org/joomla-cms/10363</a>.</sub>

**Comment by zero-24** [comment]
Thanks. RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10363">issues.joomla.org/joomla-cms/10363</a>.</sub>

**Pull request merged**